test/acceptance/tests/basic/terraform/mesh-gateway-validate/main.tf

```diff
@@ -0,0 +1,34 @@
+provider "aws" {
+  region = "us-west-2"
+}
+
+variable "kind" {
+  type = string
+}
+
+variable "consul_namespace" {
+  type = string
+}
```

Comment:
Should we remove the `consul_namespace` variable from gateway-task, since gateways always go in the default namespace?

Reply:
Yes, I think this suggestion is a better approach than having the variable and validating that it is `""` or `"default"`. I will remove it.